### PR TITLE
Fixes to make this work with Controller

### PR DESCRIPTION
**Pull request by HichamMourad**
Made a few fixes to make this work with Automation controller.  Also needed to update the Postgres version.

### CODE REVIEW

**Comment by HichamMourad** [comment]
Fixed to work with AAP on AWS.

File: playbooks/webapp/tasks/create.yaml [hunk 1]
```diff
@@ -43,6 +43,10 @@
       ansible.builtin.set_fact:
         shared_az: "{{ region_av_zones[0] }}"
 
+    - name: Print the value of Subnet_cidr
```
Comment (jillr) [inline playbooks/webapp/tasks/create.yaml:46]: I don't think this debug task is needed.

**Review by jillr — CHANGES_REQUESTED** [review]
Thanks for this @HichamMourad. One small item, otherwise this lgtm.

**Comment by jillr** [comment]
This PR will also need a [changelog](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst)

File: playbooks/webapp/tasks/create.yaml (hunk 1 above)
Comment (HichamMourad) [reply playbooks/webapp/tasks/create.yaml]: made the change.  Thanks.

**Comment by jillr** [comment]
@HichamMourad can you please add a [changelog](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst)?  The milestone sanity failures are unrelated to this change, I'll make sure we have a ticket in the backlog to get that addressed for this repo (and the other validated content repos).

**Pull request closed without merging**